src/main/scala/devices/uart/UART.scala

```diff
@@ -18,16 +19,21 @@ case class UARTParams(
   oversample: Int = 4,
   nSamples: Int = 3,
   nTxEntries: Int = 8,
-  nRxEntries: Int = 8)
+  nRxEntries: Int = 8,
+  wire4: Boolean = false,
```

Comment:
Can these names be more descriptive/less abbreviated? I might suggest:
-includeFourWire
-includeParity
-include???Parity (I'm not sure what `ind` is)